dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/HvdcVSC/BaseControls/DCVoltageControl/BaseDCVoltageControl.mo

```diff
@@ -59,13 +59,15 @@ model BaseDCVoltageControl "Base DC Voltage Control for the HVDC VSC model"
     Placement(visible = true, transformation(origin = {-110, 6}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
   Modelica.Blocks.Math.Gain deltaVDCCalc(k = RdcPu * SNom / SystemBase.SnRef) annotation(
     Placement(visible = true, transformation(origin = {-50, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
-  Modelica.Blocks.Continuous.FirstOrder firstOrder(T = 1e-4, y_start = Ip0Pu) annotation(
```

Comment:
Why have you removed it ? Is it not necessary anymore ?

Reply:
it is not used in Eurostag.